### ISSUE TITLE
Shall we archive this repo?

### ISSUE THREAD

**Issue by thanodnl** [implicit-explicit/meetup_hacking#9]
@mrmrcoleman cc @nustiueudinastea  @lalyos 

Recently I have been receiving security emails around this project, putting this repo back into my attention.

Given our ambitious start 8 years ago, but little recent interaction, I wonder how you are feeling about archiving this repo. The code will still be open and readable, but it gives a good signal it is not maintained. This should also stop the weekly security emails surrounding the project.

I don't think I have the powers to archive, I think these are with @mrmrcoleman , hence this reachout.

Documentation on archiving is here: https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories
TLDR: https://github.com/implicit-explicit/meetup_hacking/settings << scroll to bottom of the page, press `Archive this repository` in the Danger Zone.
